docs/docsite/rst/getting_started/get_started_playbook.rst

```diff
@@ -0,0 +1,69 @@
+.. _get_started_playbook:
+
+*******************
+Creating a playbook
+*******************
+
+Playbooks are automation blueprints, in ``YAML`` format, that Ansible uses to deploy and configure managed nodes.
+Playbooks define the order in which Ansible performs actions, from top to bottom, and contain the following objects:
+
+Play
+   An ordered list of tasks that maps to managed nodes in an inventory.
+
+Task
+   A list of one or more modules that defines the actions that Ansible performs.
```

Comment:
`actions` is an overloaded term to use here - action plugins run on the control node. But besides that, each task is just one module/action plugin. Since the Play definition is a list of tasks it should be clear multiple modules can be used.

Reply:
How about s/actions/operations?